model-zoo/src/main/java/ai/djl/basicmodelzoo/cv/classification/MobileNetV2.java

```diff
@@ -0,0 +1,410 @@
+                        builder.multiTimes[2],
+                        builder.batchNormMomentum);
+        Block bottleNeck4 =
+                makeStage(
+                        builder.repeatTimes[4],
+                        builder.filters[3],
+                        builder.filters[4],
+                        builder.strides[4],
+                        builder.multiTimes[3],
+                        builder.batchNormMomentum);
+        Block bottleNeck5 =
+                makeStage(
+                        builder.repeatTimes[5],
+                        builder.filters[4],
+                        builder.filters[5],
+                        builder.strides[5],
+                        builder.multiTimes[4],
+                        builder.batchNormMomentum);
+        Block bottleNeck6 =
+                makeStage(
+                        builder.repeatTimes[6],
+                        builder.filters[5],
+                        builder.filters[6],
+                        builder.strides[6],
+                        builder.multiTimes[5],
+                        builder.batchNormMomentum);
+        Block bottleNeck7 =
+                makeStage(
+                        builder.repeatTimes[7],
+                        builder.filters[6],
+                        builder.filters[7],
+                        builder.strides[7],
+                        builder.multiTimes[6],
+                        builder.batchNormMomentum);
+
+        SequentialBlock conv1 = new SequentialBlock();
+        for (int i = 0; i < builder.repeatTimes[8]; i++) {
+            conv1.add(
+                            Conv2d.builder()
+                                    .setKernelShape(new Shape(1, 1))
+                                    .setFilters(builder.filters[8])
+                                    .optStride(new Shape(builder.strides[8], builder.strides[8]))
+                                    .optBias(false)
+                                    .build())
+                    .add(
+                            BatchNorm.builder()
+                                    .optEpsilon(2E-5f)
+                                    .optMomentum(builder.batchNormMomentum)
+                                    .build())
+                    .add(Activation.reluBlock());
+        }
+
+        Block conv2 =
+                Conv2d.builder()
+                        .setKernelShape(new Shape(1, 1))
+                        .setFilters((int) builder.outSize)
+                        .optBias(false)
+                        .build();
+        return mobileNet
+                .add(pre)
+                .add(bottleNeck1)
+                .add(bottleNeck2)
+                .add(bottleNeck3)
+                .add(bottleNeck4)
+                .add(bottleNeck5)
+                .add(bottleNeck6)
+                .add(bottleNeck7)
+                .add(conv1)
+                .add(Pool.globalAvgPool2dBlock())
+                .add(
```

Comment:
Here and on 265, you can use `SequentialBlock.addSingleton`. This lets you simplify your code a bit, but also quite importantly will reduce all of these levels of indentations

Reply:
got it!